ngrok-java/src/main/java/com/ngrok/Session.java

```diff
@@ -60,15 +60,15 @@ public default LabeledTunnel labeledTunnel() throws IOException {
     public LabeledTunnel labeledTunnel(LabeledTunnel.Builder builder) throws IOException;
 
     public interface StopCallback {
-        public void stop();
+        public void onStop();
```

Comment:
I wonder about the renaming here. These are supposed to tell the agent to stop/restart/update, they are not so much events, but commands. How the application reacts on them is up to itself tho.

Reply:
Ah, I see. Still seems valuable to indicate that it's coming _to_ the agent with `on`. 

How about `onStopCommand` etc?